Controller/Controller.php

```diff
@@ -73,7 +73,7 @@ class Controller
      */
     public function __construct(
         TranslatorInterface $translator,
-        Twig_Environment $twig,
+        Environment $twig,
```

Comment:
I think without type hint, you can keep the 3.4 compatibility. Take a look here : https://github.com/nelmio/NelmioApiDocBundle/pull/1560/files

Reply:
Nice solution, I implemented it to restore Symfony 2 and 3 support.

Should we maintain 2.8 support? It has become unmaintained as of november 2019